proof_of_concept/asset_store_server/run_api.py

```diff
@@ -0,0 +1,77 @@
+"""Run asset_store_server api."""
+import logging
+import threading
+
+import connexion
+from flask import Flask
+from flask_injector import FlaskInjector
+
+from proof_of_concept.asset_store_server.store import AssetStore
+
+logger = logging.getLogger(__file__)
+
+
+def _configure_dependency_injection(flask_app: Flask, store: AssetStore
```

Comment:
For me this is easiest to setup because I am used to dependency injection pattern, but I am curious whether you find this confusing.